account_analytic_no_lines/views/account_move_line.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+<record id="view_move_line_form" model="ir.ui.view">
+    <field name="name">hide_analytic_lines.account.move.line.form</field>
+    <field name="model">account.move.line</field>
+    <field name="inherit_id" ref="account.view_move_line_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='analytic_line_ids']/.." position="attributes">
```

Comment:
Directly use `field` here

Reply:
This xpath doesn't target the field but the group just before (which has no name).